lib/tower_slack/reporter.ex

```diff
@@ -74,9 +74,11 @@ defmodule TowerSlack.Reporter do
   end
 
   defp post_message(id, similarity_id, kind, reason, stacktrace \\ []) do
-    {:ok, _} =
-      TowerSlack.Message.new(id, similarity_id, kind, reason, stacktrace)
-      |> TowerSlack.Client.deliver()
+    message = TowerSlack.Message.new(id, similarity_id, kind, reason, stacktrace)
+
+    async(fn ->
+      {:ok, _} = TowerSlack.Client.deliver(message)
+    end)
```

Comment:
Given tower 0.8 no longer calls us (the reporter) from an async task we
should have async whatever it is long running task, like the network
request.